assets/css/src/amp-customizer.css

```diff
@@ -1,5 +1,54 @@
-#customize-controls .customize-info .preview-notice {
+#customize-controls #customize-info .preview-notice {
 	display: block;
 	line-height: 1.4;
 	margin-right: 28px;
 }
+
+#customize-theme-controls .control-section-amp_active_theme_settings_import > .accordion-section-title {
+	cursor: default;
+}
+
+#customize-theme-controls .control-section-amp_active_theme_settings_import > .accordion-section-title::after {
+	content: "";
+}
+
+#customize-theme-controls .control-section-amp_active_theme_settings_import > .accordion-section-title > button {
+	float: right;
+	font-weight: normal;
+}
+
+#customize-controls .control-section-amp_active_theme_settings_import > .accordion-section-title,
+#customize-controls .control-section-amp_active_theme_settings_import > .accordion-section-title:hover {
+	cursor: default;
+	background: #fff;
+	color: #555d66;
+	border-top: 1px solid #ddd;
+	border-bottom: 1px solid #ddd;
+	border-left: none;
+	border-right: none;
+	padding: 12px;
+	font-weight: normal;
+	margin: 0 0 15px 0;
+}
+
+#customize-controls .control-section-amp_active_theme_settings_import > .accordion-section-title details {
+	margin: 5px 0;
+}
+
+#customize-controls .control-section-amp_active_theme_settings_import > .accordion-section-title summary {
+	cursor: pointer;
+	display: inline-block;
```

Comment:
The summary marker is not being shown in Firefox:

![image](https://user-images.githubusercontent.com/16200219/88984679-c4c33200-d2bd-11ea-9658-9afab8253f36.png)

 It seems if the `display` rule for the `summary` element does not match its parent's, the marker is not shown.

```suggestion
	display: list-item;
```